Source/WebCore/html/parser/HTMLConstructionSite.cpp

```diff
@@ -517,7 +517,8 @@ void HTMLConstructionSite::insertHTMLFormElement(AtomHTMLToken&& token, bool isD
     // form element pointer to point to the element created.
     if (!openElements().hasTemplateInHTMLScope())
         m_form = &formElement;
-    formElement.setDemoted(isDemoted);
+    if (isDemoted)
+        ownerDocumentForCurrentNode();
```

Comment:
We can just remove this code. It seems like the Chromium is just using the document for an use counter, which we don't have.

We should also remove the `isDemoted` parameter from this function and its callsites.

Reply:
Done.